texttospeech/ssml_addresses/tts.py

```diff
@@ -0,0 +1,127 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START tts_ssml_address_imports]
+from google.cloud import texttospeech
+
+# For Python 3, instead use:
+# import html
+import cgi
+# [END tts_ssml_address_imports]
+
+
+# [START tts_ssml_address_audio]
+def ssml_to_audio(ssml_text, outfile):
+    # Generates SSML text from plaintext.
+    #
+    # Given a string of SSML text and an output file name, this function
+    # calls the Text-to-Speech API. The API returns a synthetic audio
+    # version of the text, formatted according to the SSML commands. This
+    # function saves the synthetic audio to the designated output file.
+    #
+    # Args:
+    # ssml_text: string of SSML text
+    # outfile: string name of file under which to save audio output
+    #
+    # Returns:
+    # nothing
+
+    # Checks to make sure ssml_text is not empty
+    if ssml_text is None:
+        print('ssml_to_audio() error. The input text does not exist.')
+        return
+
+    # Instantiates a client
+    client = texttospeech.TextToSpeechClient()
+
+    # Sets the text input to be synthesized
+    synthesis_input = texttospeech.types.SynthesisInput(ssml=ssml_text)
+
+    # Builds the voice request, selects the language code ("en-US") and
+    # the SSML voice gender ("MALE")
+    voice = texttospeech.types.VoiceSelectionParams(
+        language_code='en-US',
+        ssml_gender=texttospeech.enums.SsmlVoiceGender.MALE)
+
+    # Selects the type of audio file to return
+    audio_config = texttospeech.types.AudioConfig(
+        audio_encoding=texttospeech.enums.AudioEncoding.MP3)
+
+    # Performs the text-to-speech request on the text input with the selected
+    # voice parameters and audio file type
+    response = client.synthesize_speech(synthesis_input, voice, audio_config)
+
+    # Writes the synthetic audio to the output file.
+    with open(outfile, 'wb') as out:
+        out.write(response.audio_content)
+        print('Audio content written to file ' + outfile)
+    # [END tts_ssml_address_audio]
+
+
+# [START tts_ssml_address_ssml]
+def text_to_ssml(inputfile):
+    # Generates SSML text from plaintext.
+    # Given an input filename, this function converts the contents of the text
+    # file into a string of formatted SSML text. This function formats the SSML
+    # string so that, when synthesized, the synthetic audio will pause for two
+    # seconds between each line of the text file. This function also handles
+    # special text characters which might interfere with SSML commands.
+    #
+    # Args:
+    # inputfile: string name of plaintext file
+    #
+    # Returns:
+    # A string of SSML text based on plaintext input
+
+    # Parses lines of input file
+    try:
+        with open(inputfile, 'r') as f:
```

Comment:
You can probably drop the `try` and `except`
Just let the default IOError be thrown since you'll be exiting immediately.